examples/seismic/abc_methods/01_introduction.ipynb

```diff
@@ -0,0 +1,1192 @@
+      "text/plain": [
+       "<Figure size 1152x576 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "graph2dvel(v0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To build the *temporal grid* we use the command *TimeAxis*. Before using  *TimeAxis*, we need to set the following parameters:\n",
+    "\n",
+    "- *t0* is the initial time simulation in milliseconds (ms);\n",
+    "- *tn* is the final time simulation in milliseconds (ms);\n",
+    "- *CFL* is the numerical value that imposes a restriction on the size of the time step, in order to obtain stability of the method;\n",
+    "- *vmax* denotes the maximum velocity propagation. This value is obtained by analyzing the maximum value of the *vel* array;\n",
+    "- *dtmax* is the upper limit for $\\Delta t$. This value is constructed from the *hx*, *hy* and *vmax* values.\n",
+    "- *ntmax* is the number of time steps required to reach time $t_ {F}$;\n",
+    "- dt0 is the value of$\\Delta t$ that can be used in the numerical method and that guarantees its stability;\n",
+    "\n",
+    "The parameters mentioned above are given by:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "t0    = 0.\n",
+    "tn    = 1000.   \n",
+    "CFL   = 0.4\n",
+    "vmax  = np.amax(v0) \n",
+    "dtmax = np.float64((min(hx,hz)*CFL)/(vmax))\n",
```

Comment:
Why `float64`?

Reply:
To have more precision in the calculation of the fields of interest. Is it necessary or mandatory to use the float32 format ?!

Reply:
@rhodrin Here is it ok?!